Formula/jupyter-julia.rb

```diff
@@ -0,0 +1,251 @@
+      tag:      "v2.7.2",
+      revision: "d4e8a6dbb2341d5c282134c08da8989ca06f71dd"
+  end
+
+  resource "PrecompileTools" do
+    url "https://github.com/JuliaLang/PrecompileTools.jl.git",
+      tag:      "v1.1.2",
+      revision: "11410b2970041907c2396d374ad7893d6f89a118"
+  end
+
+  resource "Preferences" do
+    url "https://github.com/JuliaPackaging/Preferences.jl.git",
+      tag:      "v1.4.0",
+      revision: "e8082bc48a40dc27ed427e5ac9d17398d9949a14"
+  end
+
+  resource "MbedTLS" do
+    url "https://github.com/JuliaLang/MbedTLS.jl.git",
+      tag:      "v1.1.7",
+      revision: "fc1d4320b327df033d4aa741410170d93cef544c"
+  end
+
+  resource "Conda" do
+    url "https://github.com/JuliaPy/Conda.jl.git",
+      tag:      "v1.9.1",
+      revision: "79c5487dab36bb68ab61f8e315f61ce466cb16ef"
+  end
+
+  resource "VersionParsing" do
+    url "https://github.com/JuliaInterop/VersionParsing.jl.git",
+      tag:      "v1.3.0",
+      revision: "027e425d5d66166698ee66a1cd111bf6f2c7ba7b"
+  end
+
+  resource "SoftGlobalScope" do
+    url "https://github.com/JuliaLang/SoftGlobalScope.jl.git",
+      tag:      "v1.1.0",
+      revision: "68195b7ed749832c1f54ccb3dcbfce0d317c8f66"
+  end
+
+  resource "ZMQ" do
+    url "https://github.com/JuliaInterop/ZMQ.jl.git",
+      tag:      "v1.2.1",
+      revision: "cf2b7a481346c447edea593c917c160d2e843764"
+  end
+
+  resource "ZeroMQ_jll" do
+    url "https://github.com/JuliaBinaryWrappers/ZeroMQ_jll.jl.git",
+      tag:      "ZeroMQ-v4.3.4+0",
+      revision: "775ef7a2ac3053c0d2ecfe05e26fb87b95368c4d"
+  end
+
+  resource "JLLWrappers" do
+    url "https://github.com/JuliaPackaging/JLLWrappers.jl.git",
+      tag:      "v1.4.1",
+      revision: "bade7d4b7af3e1c0c590bfd383a3df5ed10852f8"
+  end
+
+  resource "libsodium_jll" do
+    url "https://github.com/JuliaBinaryWrappers/libsodium_jll.jl.git",
+      tag:      "libsodium-v1.0.20+0",
+      revision: "d7881f6bf029bae13dfddb7e18f6a975601d82b1"
+  end
+
+  # above deps are for IJulia, below deps are for LanguageServer
+
+  resource "LanguageServer" do
+    url "https://github.com/julia-vscode/LanguageServer.jl.git",
+      tag:      "v4.4.0",
+      revision: "8a3ae73b510aa66e302f57d5d9545a550fcb4e7a"
+  end
+
+  resource "CSTParser" do
+    url "https://github.com/julia-vscode/CSTParser.jl.git",
+      tag:      "v3.3.6",
+      revision: "2831264262b048a83f84a7b5dc54338e7cf96ee7"
+  end
+
+  resource "Tokenize" do
+    url "https://github.com/JuliaLang/Tokenize.jl.git",
+      tag:      "v0.5.24",
+      revision: "49b112248e846040c13f28e223493f90cc717fa5"
+  end
+
+  resource "URIs" do
+    url "https://github.com/JuliaWeb/URIs.jl.git",
+      tag:      "v1.5.0",
+      revision: "4fd45153c0aac8094877d9a703bc51d1b9aa9c00"
+  end
+
+  resource "TestItemDetection" do
+    url "https://github.com/julia-vscode/TestItemDetection.jl.git",
+      # version 0.2.0 exists in the repo, but there is no Git tag...
+      # tag:      "v0.2.0",
```

Comment:
do we want to log an upstream issue for this?

Reply:
Done: https://github.com/julia-vscode/TestItemDetection.jl/issues/14

Reply:
👍